src/index.html

```diff
@@ -1,13 +1,356 @@
+    <footer class="footer">
+      <section class="footer-top">
+        <div class="footer-top__subscribe subscribe">
+          <h2 class="subscribe__header">Підписка</h2>
+          <p class="subscribe__text">
+            Бути в курсі всіх актуальних подій та останніх новин.
+          </p>
+          <div class="subscribe__field field">
+            <input
+              class="field__email email"
+              type="text"
+              placeholder="E-mail"
+            >
+            <a class="ticket__button ticket__button--secondary">
+              <img
+                class="ticket__svg"
+                src="./images/Vector.svg"
+                alt=""
+                >
+            </a>
+          </div>
+        </div>
+
+      </section>
+
+      <section class="footer-bottom container inner-padding">
+        <div class="footer-bottom__left-side left-side">
+          <div class="left-side__icons icons">
+            <a href="http://namu.kiev.ua/">
+              <img
+              class="icons__namu"
+              src="./images/NA MU.svg"
+              alt=""
+            >
+            </a>
+            <div class="icons__network">
+              <a href="https://uk-ua.facebook.com/">
+                <img
+                  class="icons__facebook"
+                  src="./images/facebook.svg"
+                  alt="facebook"
+                >
+              </a>
+              <a href="https://www.instagram.com/">
+                <img
+                  class="icons__instagram"
+                  src="./images/instagram.svg"
+                  alt="instagram"
+                >
+              </a>
+            </div>
+
+          </div>
+
+
+          <div class="shedule shedule--padding">
+            <p class="shedule__header">Графік роботи</p>
+
+            <p class="shedule__days">
+              <span>12:00 — 20:00</span> середа<br>
+              <span>12:00 — 20:00</span> четвер<br>
+              <span>12:00 — 20:00</span> п’ятниця<br>
+              <span>11:00 — 19:00</span> субота<br>
+              <span>11:00 — 19:00</span> неділя<br>
+              <br>
+              Вихідні понеділок,<br>
+              вівторок<br>
+              Каса припиняє роботу <br>
+              за 1 годину<br>
+              до закриття музею
+
+            </p>
+
+
+
+          </div>
+          <div class="shedule">
+            <p class="shedule__header">Контакти:</p>
+
+            <p class="shedule__days">
+              вул. М. Грушевського, 6; Київ, <br>
+              01001 <br>
+              тел. +38 (044) 278-13-57<br>
+              факс +38 (044) 278-74-54<br>
+              e-mail: info@namu.kiev.ua<br>
+            </p>
+          </div>
+        </div>
+
+        <div class="footer-bottom__right-side right-side">
+          <div class="right-side__tabs tabs">
+            <ul class="tabs__list list">
+
+              <li class="list__item item">
+                <a class="item__link link" href="">Галерея</a>
```

Comment:
Не залишай href пустим
```suggestion
                <a class="item__link link" href="">
                  Галерея
                </a>
```